File: app/src/main/java/com/commonpepper/photosen/database/PhotosenDatabase.java
package com.commonpepper.photosen.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.commonpepper.photosen.model.Photo;

@Database(entities = {Photo.class}, version = 1)
public abstract class PhotosenDatabase extends RoomDatabase {
    public abstract HistoryDao getHistoryDao();
}

File: app/src/main/java/com/commonpepper/photosen/database/PhotosenDatabase.kt
package com.commonpepper.photosen.database

import androidx.room.Database
import androidx.room.RoomDatabase
import com.commonpepper.photosen.model.Photo

@Database(entities = [Photo::class], version = 2)
abstract class PhotosenDatabase : RoomDatabase() {
    abstract val historyDao: HistoryDao
}
